Packs/GigamonThreatINSIGHT/Integrations/GigamonThreatINSIGHT/GigamonThreatINSIGHT.py

```diff
@@ -400,6 +401,80 @@ def formatEvents(r_json):
     return newData
 
 
+def getFirstFetch(firstFetch) -> dict[str, Any]:
+
+    pieces = firstFetch.strip().split()
+    unit = pieces[len(pieces) - 1]
+
+    try:
+        delta = arg_to_number(
+            arg=pieces[0],
+            arg_name='first_fetch',
+            required=False
+        )
+    except Exception:
+        delta = 0
+
+    days: int = delta if delta and unit == 'days' else 0
+    hours: int = delta if delta and unit == 'hours' else 0
+
+    if days == 0 and hours == 0:
```

Comment:
This needs to be mandatory in UI, else the container for the integration does not start up.

Reply:
There is a default value of 7 days. It was put as '7' instead of '7 days' in the yml but it should work. what was the error in the logs?